.golangci.yml

```diff
@@ -70,3 +69,20 @@ issues:
     - linters:
         - wsl
       text: "only one cuddle assignment allowed before range statement"
+
+    # Ignore new line warning at L30. New line is added to reduce line length according to lll
+    - linters:
+        - stylecheck
+      text: "ST1005: "
+      path: pkg/networkplugin-syncer/handlers/ovn/handler.go
```

Comment:
The error seems like a false-positive bug with the linter. We should ignore this one line in code via:

`//nolint:stylecheck This seems like a false-positive bug with the linter - the actual string has no newline`

Reply:
ahh yes...thats a good alternative. Forgot this one